src/App.jsx

```diff
@@ -4,11 +4,14 @@ import Home from "./pages/home.jsx"
 import Symptome from "./pages/symptome.jsx"
 import Medikamente from "./pages/medikamente.jsx"
 import Termine from "./pages/termine.jsx"
+import NeuerTermin from "./pages/neuertermin.jsx"
+import NeuesMedikament from "./pages/neuesmedikament.jsx"
```

Comment:
When importing jsx files, you don't necessarily have to use the suffix jsx. But this is just a subtle hint.

Reply:
Thanks!! :-)